### ISSUE TITLE
Syntax highlighting does not work immediately

### ISSUE THREAD

**Issue by LennDG** [pfeiferj/vscode-hurl#8]
After installing the Hurl extension in vscode 1.81 there is no syntax highlighting for `.hurl` files.
When trying another extension (hurl-syntax-highlightning) it immediately provides highlighting.

Example file that has no highlighting:
```
GET http://www.sample.net

HTTP 200
```

Is there an additional setting that needs to be set to provide highlighting?

**Comment by pfeiferj** [comment]
There shouldn't be anything required. I just tried it out in v1.81.1 and it seems to be working. What os and architecture are you on? In theory the extension should be universal but there might be something I've missed depending on the environment

**Comment by pfeiferj** [comment]
> Is there an additional setting that needs to be set to provide highlighting?

There shouldn't be anything required. I just tried it out in v1.81.1 and it seems to be working. What os and architecture are you on? In theory the extension should be universal but there might be something I've missed depending on the environment

**Comment by LennDG** [comment]
This is on Ubuntu 22.04. I will check it on my Windows PC tomorrow. As far as I can tell it should for sure be working out of the box, so I will keep digging a bit.

**Comment by Esequiel378** [comment]
Hello there! I've just tested it in Windows 11 with Hurl installed, and it does not have syntax highlighting.

The file name is `organizations.hurl`

![organizations.hurl](https://github.com/pfeiferj/vscode-hurl/assets/40343304/86b5c625-a6ca-42a4-b50b-ee59e203fa19)

**Comment by elliotlarson** [comment]
Yeah, I'm experiencing the same thing on Mac OS Ventura.  I'm not getting any syntax highlighting.

**Comment by williamd1k0** [comment]
I don't know exactly why, but for me it works fine on VSCode (Flatpak), but not on VSCodium (AUR). I'm using Manjaro Linux.

**Comment by jupegarnica** [comment]
Same here, not syntax highlighting,  nor in windows 11 nor in ubuntu 22.04 

code --version
1.86.1
31c37ee8f63491495ac49e43b8544550fbae4533
x64

vscode hurl 
v1.3.0

**Comment by j127** [comment]
Syntax highlighting isn't working here either. 

- M1 Mac, Sonoma 14.5
- hurl 4.3.0 (/opt/homebrew/bin/hurl)

VS Code data:

Version: 1.92.1 (Universal)
Commit: eaa41d57266683296de7d118f574d0c2652e1fc4
Date: 2024-08-07T20:16:39.455Z
Electron: 30.1.2
ElectronBuildId: 9870757
Chromium: 124.0.6367.243
Node.js: 20.14.0
V8: 12.4.254.20-electron.0
OS: Darwin arm64 23.5.0

**Comment by pfeiferj** [comment]
best guess is that it's something weird going on with your file associations. Ensure on the bottom right where it says what language the file is that it says hurl. If not you might need to click that and setup the language association. The plugin adds the hurl language type with an association to .hurl files but if it's been overridden manually or by another plugin it may not autodetect correctly

**Comment by mt-ib** [comment]
Not working for me either, I've verified file association is set to hurl. Are there any logs I could dig out to help troubleshoot this further?

**Comment by pfeiferj** [comment]
> Not working for me either, I've verified file association is set to hurl. Are there any logs I could dig out to help troubleshoot this further?

Click on the extension inside vscode then click the features tab and finally click on runtime status, and if there's an error that prevents it from working it should show up there
